src/app/api/admin/services/vizolv/route.ts

```diff
@@ -0,0 +1,67 @@
+import db from '@/db';
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function GET(req: NextRequest) {
+  const authKey = req.headers.get('Authorization');
+  const page = parseInt(req.nextUrl.searchParams.get('page') || '0');
```

Comment:
remove pagination

Reply:
Removed pagination. Added logic to fetch by Id